source/extensions/filters/common/ext_authz/ext_authz_grpc_impl.cc

```diff
@@ -138,6 +138,23 @@ void GrpcClientImpl::toAuthzResponseHeader(
   }
 }
 
+Grpc::RawAsyncClientSharedPtr AsyncClientCacheImpl::getOrCreateAsyncClient(
+    const envoy::extensions::filters::http::ext_authz::v3::ExtAuthz& proto_config) {
+  // The cache only stores google Envoy which need to preserve channels.
```

Comment:
nit: I'm not sure I understand this comment. Shouldn't it be "The cache stores a google rpc client..."?

Reply:
Changed